challenges/functions.js

```diff
@@ -6,17 +6,28 @@
   * The last parameter accepts a callback 
   * In the body of the function return the callback with the two parameters that you created
 */
-
+let  Func = (param1,param2,cb) => {
+  return cb(param1 + param2)
+}
 
 /* Step 2: Create several functions to callback with consume();
   * Create a function named add that returns the sum of two numbers
   * Create a function named multiply that returns the product of two numbers 
   * Create a function named greeting that accepts a first and last name and returns "Hello first-name last-name, nice to meet you!"
 */
+let func = (param1, param2, cb) => {
+  return cb(param1 + param2)
+}
 
+let tuck = (param1, param2, cb) => {
+return cb(param1 * param2)
+}
+let greeting = (first,last) => {
+  return `Hello ${this.first} ${this.last}, nice to meet you!`
+}
 
 /* Step 3: Check your work by uncommenting the following calls to consume(): */
-//consume(2,2,add); // 4
+// consume(2,2,add); // 4
 // consume(10,16,multiply); // 160
 // consume("Mary","Poppins", greeting); // Hello Mary Poppins, nice to meet you!
 
```

Comment:
* Are you currently understanding Closure and able to provide an explanation for the questions bellow?